bootstrap/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const appName = "My App";
+export const backend = "https://developer.na.gooddata.com";
+export const workspace = "";
+export const workspaceFilter: RegExp | undefined = undefined; // regexp to filter workspaces in WorkspacePicker. Example: /test/ or /^GoodData/,
```

Comment:
we could use description of these. esp. the workspace & workspaceFilter have some extra logic tied to them so would be good to give more information at this point

Reply:
Fixed.